AUTHORS

```diff
@@ -0,0 +1,14 @@
+
+All contributing authors are listed in this file below. Repository
+history available in https://github.com/MDAnalysis/membrane-curvature.
+
+Chronological list of authors
+-----------------------------
+
+2021
+  - Estefania Barreto-Ojeda
+  - Oliver Beckstein
+  - Irfan Alibay
+
+2023
+  - Lily Wang
```

Comment:
@IAlibay you contributed in 2023, should I repeat your name here as well?

Reply:
Usually the easiest thing is for author files to be "at first time of contribution", that makes life a ton easier for maintenance going forward 😅

Reply:
Gotcha, thanks for confirming.